scripts/ci_setup.py

```diff
@@ -179,6 +179,14 @@ def add_arguments(parser: ArgumentParser) -> ArgumentParser:
         help='Configurations used in the build in key=value format'
     )
 
+    parser.add_argument(
+        '--maui-version',
+        dest='maui_version',
+        default='',
+        required=False,
+        type=str,
+        help='Version of Maui to pass'
```

Comment:
Can you update this to be slightly more descriptive? Something like Version of Maui used to build app packages.